consul/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG - consul
 
+## 1.6.0 / 2018-10-12
+
+* [Added] Command to validate metadata. See [#2269](https://github.com/DataDog/integrations-core/pull/2269).
```

Comment:
This seems to be here because the metadata file was fixed in this PR. I don't think we need to include it here.